sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveInheritance.cs

```diff
@@ -57,7 +57,9 @@ public class AvoidExcessiveInheritance : ParameterLoadingDiagnosticAnalyzer
         [RuleParameter(
             key: "filteredClasses",
             type: PropertyType.String,
-            description: "Classes to be filtered out of the count of inheritance. Depth counting will stop when a filtered class is reached. (String)",
+            description: "Comma separated list of classes to be filtered out of the count of inheritance. Depth " +

```

Comment:
Nit-picking comment, feel free to ignore if it's more than 5 minutes of work.
Comma separated => Comma-separated
Ex value: => For example: